tests/bluetooth/tester/src/bttester.h

```diff
@@ -630,6 +630,9 @@ void tester_init(void);
 void tester_rsp(u8_t service, u8_t opcode, u8_t index, u8_t status);
 void tester_send(u8_t service, u8_t opcode, u8_t index, u8_t *data,
 		 size_t len);
+struct net_buf *btp_rp_alloc(void);
+void btp_rp(struct net_buf *buf, u8_t service, u8_t opcode, u8_t index,
```

Comment:
let be consistent and use rsp instead of rp in function names eg btp_send_rsp()

Reply:
done